extensions/amp-carousel/0.1/test/test-scrollable-carousel.js

```diff
@@ -347,7 +375,11 @@ describes.realWin(
           impl.goCallback(1, /*animate*/ false);
           impl.goCallback(-1, /*animate*/ false);
 
-          const updateInViewportSpy = sandbox.spy(impl, 'updateInViewport');
+          const ownerImpl = Services.ownersForDoc(impl.element);
+          const updateInViewportSpy = sandbox.spy(
```

Comment:
move this to beforeEach to avoid code duplications

Reply:
The problem is that currently everything is constructed after beforeEach (https://github.com/ampproject/amphtml/blob/1264bc6d75844176af61d954c1428cc47b3805c1/extensions/amp-carousel/0.1/test/test-scrollable-carousel.js#L130).

We could refactor these tests so there wouldn't as many duplicates as we wish, but maybe it is better left on a separate PR.

Reply:
the services installation is done in `describes.realWin`, owners service should be available already in beforeEach. Did you see it not working?